src/backlight/backlight.c

```diff
@@ -335,7 +360,7 @@ static int run(int argc, char *argv[]) {
         log_setup_service();
 
         if (argc != 3)
-                return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "This program requires two arguments.");
+                return help();
```

Comment:
No, that is wrong. The program should print help only when `--help` is specified. When called with a wrong number of arguments, it should just print the message and return an error (as before).